sdk/sdk-e2e/tests/refinanceAaveV3SparkAnyPair.test.ts

```diff
@@ -79,15 +85,25 @@ describe.skip('Refinance AaveV3 Spark | SDK', () => {
     })
 
     // Tokens
-    const debtToken: Maybe<IToken> = await chain.tokens.getTokenBySymbol({
-      symbol: config.debtTokenSymbol,
+    const sourceDebtToken: Maybe<IToken> = await chain.tokens.getTokenBySymbol({
+      symbol: config.source.debtTokenSymbol,
     })
-    assert(debtToken, `${config.debtTokenSymbol} not found`)
+    assert(sourceDebtToken, `${config.source.debtTokenSymbol} not found`)
 
-    const collateralToken: Maybe<IToken> = await chain.tokens.getTokenBySymbol({
-      symbol: config.collateralTokenSymbol,
+    const sourceCollateralToken: Maybe<IToken> = await chain.tokens.getTokenBySymbol({
+      symbol: config.source.collateralTokenSymbol,
     })
-    assert(collateralToken, `${config.collateralTokenSymbol} not found`)
+    assert(sourceCollateralToken, `${config.source.collateralTokenSymbol} not found`)
+
+    const targetDebtToken: Maybe<IToken> = await chain.tokens.getTokenBySymbol({
+      symbol: config.target.debtTokenSymbol,
+    })
+    assert(targetDebtToken, `${config.target.debtTokenSymbol} not found`)
+
+    const targetCollateralToken: Maybe<IToken> = await chain.tokens.getTokenBySymbol({
```

Comment:
`Maybe<IToken>` this explicit type declaration is not necessary and can be omitted

Reply:
The tests were done as a kind of tutorial, so there are several of those sprinkled across the different E2E tests. If you are OK with it I'd like to leave it, but I agree we don't need to do it in the actual code